frontend/src/components/PresentationExList.vue

```diff
@@ -212,10 +232,12 @@ export default {
         {
           text: this.$t("component.presentationExList.table.label"),
           value: "label",
+          sortable: false,
         },
         {
           text: this.$t("component.presentationExList.table.role"),
           value: "role",
+          sortable: false,
```

Comment:
does order by role not work?

Reply:
yes it works, just forgot to re-enable it again.